qsub/qsublist.m

```diff
@@ -170,6 +170,7 @@
             retval = 1;
           else
             retval = strcmp(strtrim(jobstatus) ,'C');
+            retval = retval | findstr(strtrim(jobstatus), 'Unknown Job Id');
```

Comment:
oh, I just figured out that in the remote branch I cannot edit. 

@octomike can you please change it into  this?
```
retval = retval | contains(jobstatus, 'Unknown Job Id');
```

Reply:
opted for `~isempty(strfind(jobstatus, 'Unknown Job Id'));` because contains is not available for 2014.. 

the `isempty()` seemed necessary, because..

```
>> logical([]) 

ans =

     []
```

:scream:

Reply:
ok, fine!